[2024-08-31T13:44:35.799+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-31T13:44:35.866+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_dag.load2 scheduled__2024-08-30T00:00:00+00:00 [queued]>
[2024-08-31T13:44:35.885+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_dag.load2 scheduled__2024-08-30T00:00:00+00:00 [queued]>
[2024-08-31T13:44:35.886+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-31T13:44:35.965+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): load2> on 2024-08-30 00:00:00+00:00
[2024-08-31T13:44:36.031+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=220) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-31T13:44:36.047+0000] {standard_task_runner.py:72} INFO - Started process 233 to run task
[2024-08-31T13:44:36.068+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'generate_dag', 'load2', 'scheduled__2024-08-30T00:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/mydag.py', '--cfg-path', '/tmp/tmp8mzs8hl2']
[2024-08-31T13:44:36.106+0000] {standard_task_runner.py:105} INFO - Job 90: Subtask load2
[2024-08-31T13:54:32.963+0000] {cli_action_loggers.py:177} WARNING - Failed to log action (psycopg2.OperationalError) connection to server at "postgres" (172.18.0.3), port 5432 failed: server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.

(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2024-08-31T13:54:35.576+0000] {job.py:229} INFO - Heartbeat recovered after 600.01 seconds
[2024-08-31T13:54:35.764+0000] {task_command.py:467} INFO - Running <TaskInstance: generate_dag.load2 scheduled__2024-08-30T00:00:00+00:00 [running]> on host 763d90797398
[2024-08-31T13:54:52.004+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='generate_dag' AIRFLOW_CTX_TASK_ID='load2' AIRFLOW_CTX_EXECUTION_DATE='2024-08-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-30T00:00:00+00:00'
[2024-08-31T13:54:52.011+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-08-31T13:54:52.012+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-08-31T13:54:52.013+0000] {logging_mixin.py:190} INFO - Current task name:load2 state:running start_date:2024-08-31 13:44:35.867251+00:00
[2024-08-31T13:54:52.014+0000] {logging_mixin.py:190} INFO - Dag name:generate_dag and current dag run status:running
[2024-08-31T13:54:52.015+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-31T13:54:52.016+0000] {python.py:240} INFO - Done. Returned value was: Hello dog!
[2024-08-31T13:54:52.189+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-31T13:54:52.193+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=generate_dag, task_id=load2, run_id=scheduled__2024-08-30T00:00:00+00:00, execution_date=20240830T000000, start_date=20240831T134435, end_date=20240831T135452
[2024-08-31T13:54:52.265+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-08-31T13:54:52.276+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-08-31T13:54:52.278+0000] {logging_mixin.py:190} INFO - Dag name:generate_dag queued_at:2024-08-31 13:43:41.238955+00:00
[2024-08-31T13:54:52.279+0000] {logging_mixin.py:190} INFO - Task hostname:763d90797398 operator:_PythonDecoratedOperator
[2024-08-31T13:54:52.377+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-31T13:54:52.477+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2024-08-31T13:54:52.957+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-31T13:54:52.971+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
